pkg/broker/ingress/ingress_handler.go

```diff
@@ -101,7 +101,14 @@ func (h *Handler) Start(ctx context.Context) error {
 }
 
 func (h *Handler) ServeHTTP(writer http.ResponseWriter, request *http.Request) {
+	writer.Header().Set("Allow", "POST, OPTIONS")
```

Comment:
should the spec say something about `OPTIONS` ? https://github.com/knative/specs/blob/main/specs/eventing/broker.md#ingress

Reply:
http://github.com/knative/specs/pull/25, `data-plane.md`, line 88.

Reply:
(I started working ahead of that commit landing)